site/en/models.md

```diff
@@ -34,6 +34,8 @@ of](https://github.com/OCR-D/core/blob/master/ocrd/ocrd/resource_list.yml):
 
 ```
 ocrd resmgr list-available
+# alternatively, using Docker:
+docker run --volume $PWD:/data --volume $PWD/models:/usr/local/share/tessdata --volume $PWD/models:/usr/local/share/ocrd-resources -w /data -- ocrd/all:maximum ocrd resmgr list-available
```

Comment:
This is just the `list-available` call but if you use the same call for `download ocrd-tesserocr-recognize ...`, the tesseract models will be downloaded to `$PWD/models` directly. I don't see that as particularly problematic but it breaks the organization of resources into processor-specific subfolders.

I.e. maybe this would be cleaner:

```suggestion
mkdir -p $PWD/models/ocrd-tesserocr-recognize
docker run --volume $PWD:/data --volume $PWD/models/ocrd-tesserocr-recognize:/usr/local/share/tessdata --volume $PWD/models:/usr/local/share/ocrd-resources -w /data -- ocrd/all:maximum ocrd resmgr list-available
```

Reply:
Good idea. In that spirit, since we are in full control of /usr/local/share in ocrd_all Docker images, couldn't we take it one step further and simply mount a single volume for /usr/local/share – and be done with all resource locations and configs?

Reply:
```
mkdir -p $PWD/models/ocrd-tesserocr-recognize
docker run --volume $PWD:/data --volume $PWD/models:/usr/local/share -w /data -- ocrd/all:maximum ocrd resmgr list-available
```

works for me, I changed the file accordingly